spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -0,0 +1,18 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
+class UserMailerPreview < ActionMailer::Preview
+
+  # Preview this email at http://localhost:3000/rails/mailers/user_mailer/account_activation
+  def account_activation
+    user = User.last
```

Comment:
You can have `user = User.new` with a random email to not have to get data from the database for the previews and make it easier to know what it should display.